### ISSUE TITLE
Latest version is unusable with ExLlama

### ISSUE THREAD

**Issue by iChristGit** [ParisNeo/lollms-webui#438]
When using the latest version every model i try to load with exllama il get either an error or just bad results (nonsense)

## Current Behavior
Please describe the behavior you are currently experiencing.

## Steps to Reproduce
Using latest version from scratch 
1. Step 1 - install exllama v2 binding
2. Step 2 - choose any model
4. Step 3 - try go ask a simple question

## Screenshots
![image](https://github.com/ParisNeo/lollms-webui/assets/20077386/3683ab3b-48fd-46e2-b751-2ac755dbe4e6)

**Comment by iChristGit** [comment]
![image](https://github.com/ParisNeo/lollms-webui/assets/20077386/6c200170-a1bf-4308-a487-83ad53424062)
Just fresh installed using v6.7 bat file installer, same results

**Comment by iChristGit** [comment]
Managed to fix the issue by using the latest bat file made 20 hours ago (not in release page)
Yi-34B and Deepseek coder models are not working, GPTQ seems fine

**Comment by ParisNeo** [comment]
Yes because they require me to enable the execution of remote code.
That's too risky of an operation. I prefer waiting that hugging face add this to their library.
Executing code from a model is a security risk as someone may create a virus and hide it inside a model.

If you trust them enough, just add  trust_remote_code=True to the from_pretrained when creating the tokenizer if you want to override this. It should work.
